gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -399,6 +400,7 @@ gst_tensor_src_iio_init (GstTensorSrcIIO * self)
   /** init properties */
   self->configured = FALSE;
   self->channels = DEFAULT_PROP_STRING;
+  self->custom_channel_list = DEFAULT_PROP_STRING;
```

Comment:
```custom_channel_list``` is GList. It does not incur erros because ```DEFAULT_PROP_STRING``` is NULL; however, it may mislead other developers to update ```DEFAULT_PROP_STRING``` with real string values, which will break it. It's better be written NULL instead of DEFAULT_PROP_STRING.

Reply:
Updated.